pandas/tests/frame/methods/test_isin.py

```diff
@@ -189,3 +189,16 @@ def test_isin_empty_datetimelike(self):
         tm.assert_frame_equal(result, expected)
         result = df1_td.isin(df3)
         tm.assert_frame_equal(result, expected)
+
+    def test_isin_int_df_string_search(self):
```

Comment:
these tests are ok here, but really the unit tests are in pandas/tests/test_algos.py

Reply:
going to swtich that for the right file